inception-project/inception
# #4655 - Opening a document in sidebar curation mode does not trigger auto-merge

**What's in the PR**
- Auto-merge when accessing a document in curation sidebar mode if the document has not been curated yet - however - for the moment only when curating to the curation user
- When reading annotations in another mode than exclusive-write-access, do not persist the annotations to disk e.g. after loading them from the initial CAS and upgrading

**How to test manually**
* Try using the curation sidebar, switching between documents, enabling/disabling curation mode and check if the initial merges happen as expected
* Use the monitoring page to reset the curation from time to time

**Automatic testing**

**Documentation**
